### ISSUE TITLE
Location of focused track provided for assistive technology is incorrect

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#2912]
**Describe the bug**
Location of focused track provided for assistive technology is incorrect - it should be roughly the same as the yellow outline.

**To Reproduce**
Steps to reproduce the behavior on Windows 10:
1. Start the narrator screen reader (Ctrl+Windows key+Enter).
2. Open audacity, and add a track.

**Actual behavior**
The blue outline of the focused track produced by Narrator is vertically offset from the yellow outline.
The bottom and the top of the blue outline in general pass through the waveform display, making it more difficult for visually impaired users to see the data. This obviously shouldn't be the case.

![image](https://user-images.githubusercontent.com/11887021/167810269-1cf8ce2f-285a-4f29-bda8-1d4325830ee0.png)

**Expected behavior**
The blue outline of the focused track should lie on top of the yellow outline, as it does in 3.1.3
![image](https://user-images.githubusercontent.com/11887021/167810592-86bec5ea-2e4f-43c8-a576-8636f63048c9.png)

**Additional information (please complete the following information):**
 - OS: Windows 10
 - Version: Audacity 3.2.0 alpha
 - (if relevant) Sound Device [e.g. Windows WASAPI, USB Sound card]

**Additional context**
This is a regression wrt Audacity 3.1.3

**Comment by DavidBailes** [comment]
@LWinterberg . Can this be fixed for 3.2.0? I have supplied a simple fix.

**Comment by Paul-Licameli** [comment]
I suspect this will be found to bisect to the same commit as problem #3013

**Comment by Paul-Licameli** [comment]
Which is known: https://github.com/audacity/audacity/commit/5835ae5d87f195ce0e2e8b985586b2ed45c4e1ca

**Comment by Paul-Licameli** [comment]
Thank you, David!  I overlooked the PR at first.

I can't figure out what is wrong with my GitHub settings.  I want to get a notification each time a pull request is opened, but it doesn't happen, as it does for issues.

**Comment by dozzzzer** [comment]
As can be seen from the screenshot, the selected area doesn't match the audio track section when running Audacity on Windows 11 VM in Parallels. Apparently, DPI is disregarded, that's why it happens. Since it can be fixed, I'm gonna  reopen it and lower the priority.

<img width="1176" alt="image" src="https://user-images.githubusercontent.com/70063970/177554329-90100d95-eede-43e1-975b-80e201e146bd.png">

**Comment by DavidBailes** [comment]
This has been fixed by: https://github.com/audacity/audacity/issues/2912
